models/forum.js

```diff
@@ -28,14 +28,48 @@ var ForumSchema = new Schema({
 		trim: true,
 		required: 'A thread needs the id of the poster'
 	},
-	imgurl: {
+	imgURL: {
 		type: String,
 		trim: true
 	},
-	comments: { // todo: validate inserted comments?
-		type: Array,
+	comments: {
+		type: [String],
 		default: []
-	}
+	},
+	edited: {
+        type: Boolean,
+        default: false
+    }
 });
+mongoose.model('forum', ForumSchema);
 
-mongoose.model('forum', ForumSchema);
\ No newline at end of file
+var CommentSchema = new Schema({
+    text: {
+    	type: String,
+		required: 'A comment needs to have a text',
+		trim: true
+	},
+	created: {
+        type: String,
+        default: new Date()
```

Comment:
Date of type String?

Reply:
Yeh I set everything as a String as a placeholder. Once we have a meeting with the app developers we can ask them how they like the data types so its easier for them to display the information.